### DISCUSSION TITLE
How to install OpenPNM?

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [PMEAL/OpenPNM#1809] [Q&A] [answered]
I wonder how we can install OpenPNM...

**Comment by jgostick** [comment, marked as answer]
The standard ``pip install openpnm`` works, but as of version 2.6 we **strongly** recommonend that you use ``conda install -c conda-forge openpnm``.  The latter approach will also install the [pardiso solver](https://www.pardiso-project.org/) which is *substantially* faster than the default solver that comes with scipy.  In fact the ``spsolve`` method in scipy is so slow that if your network has more than 10,000 pores, OpenPNM will raise an error and prevent you from running.  It's just slow slow that you'll think the code has hung up.

**Reply by jluleijiang** [reply]
I have downloaded anaconda and pnm2.7, but because I have no programming foundation, I don't know the specific opening steps and corresponding operations. Can you give me a simple guide? Thank you.

**Reply by jgostick** [reply]
@ma-sadeghi has written a detailed guide for installing our other package [PoreSpy](https://porespy.org/user_guide/installation.html).  The steps are the same but just replace ``porespy`` with ``openpnm``.
